### DISCUSSION TITLE
请教一下，该如何设置 ?

### DISCUSSION THREAD

**Discussion by kevzcr** [shuax/MouseInc.Settings#358] [General]
1. Mouselnc如何设置窗口 `置顶`。 我觉得软件一定可以，只是我不会。

2. Mouselnc能实现 `复制、移动到文件夹` 的快捷键吗？

**Comment by SoyKoa** [comment]
["Window", "Top"]   窗口置顶
复制移动那个使用频率不高，，没用过

**Reply by kevzcr** [reply]
看到了，现在就去试，感谢。
